stack_master.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "timecop"
   spec.add_dependency "os"
   spec.add_dependency "ruby-progressbar"
-  spec.add_dependency "commander", ">= 4.5.2", "< 5"
+  spec.add_dependency "commander", ">= 4.6.0", "< 5"
```

Comment:
Ensure StackMaster is using a version of the gem where the instance variable is named `@instance`.